### ISSUE TITLE
Live player hangs after 30 minutes

### ISSUE THREAD

**Issue by Ronileco** [kaltura/mwEmbed#1964]
It seems the Kaltura live player is hanging after 30 minutes of watching a live stream with HLS.
We can't see any prominent errors on the console or on the player other than:
[.PPAPIContext]GL ERROR :GL_INVALID_ENUM : glTexImage2D: target was GL_TEXTURE_RECTANGLE_ARB
PepperFlashPlayer.plugin: 0x29B96 is not a valid resource ID.

Which seems related to HLS plugin, chrome hardware acceleration or something like that...

Here is a test page taken from KMC:
http://video.panda-os.com/index.php/extwidget/preview/partner_id/129/uiconf_id/11604500/entry_id/0_6dzthpvi/embed/iframe?&flashvars[streamerType]=auto

Here is a test page we've integrated into our platform:
http://faithcalgary.jesusstream.com/play/Live%20Broadcast/live-broadcast-adaptive-stream/0_6dzthpvi

The player is set with live Kaltura player and the HLS module.
This doesn't happen on other player like grind we tried to use.

Did you experience such issues or can help with this?

**Comment by itaykinnrot** [comment]
no , Can you add some flash logs when the issue is reproduce?

**Comment by Ronileco** [comment]
Sadly there is no Flash messages (The debugger doesn't catch anything).
I have found out that the new v2.32 works much better, probably due to the HLS plugin fixes.

But we found that on that version the HLS doesn't play sound on Chrome.
Sound actually starts for a second and than fails.

I've upgraded the player on the link above:
http://faithcalgary.jesusstream.com/play/Live%20Broadcast/live-broadcast-adaptive-stream/0_6dzthpvi
